### ISSUE TITLE
Exempt arrow functions from requiring types

### ISSUE THREAD

**Issue by indigoviolet** [gajus/eslint-plugin-flowtype#23]
Arrow functions should be small enough that types aren't essential; it'd be nice to be able to exempt them from `flowtype/require-parameter-type` and `flowtype/require-return-type`

However, I would not want this to affect arrow functions that are set as class properties to make `bind` easier:

```
class Foo extends React.Component {
  instancemethod = (foo: Type) : ReturnType => {
     ....
  }
}

```

Is this a feasible request?

**Comment by gajus** [comment]
`flowtype/require-parameter-type` could be updated to support a setting which controls which types of functions to apply the style.

**Comment by allanhortle** [comment]
+1 on this. 

Making heavy use of chained methods with arrow functions gets really heavy handed when the same object is being passed through:

```js
return Maybe
    .fromNull(props.files)
    .filter((files: Array): Array => files.size)
    .map((files: Array): Array => {
        return files.map((file: Object): Number => file.size);
    });
```

**Comment by gajus** [comment]
Fixed in https://github.com/gajus/eslint-plugin-flowtype/pull/52 thanks to @danharper. Released in v2.6.0.
